lib/phases/cold-launch.js

```diff
@@ -86,27 +87,29 @@ ColdLaunch.prototype.capture = function() {
   this.dispatcher.on(Phase.PERFORMANCEENTRY, function(entry) {
     phase.debugEventEntry(Phase.PERFORMANCEENTRY, entry);
 
+    var perfEntry = entry;
+
     var ignore = phase.fullyLoaded &&
-        entry.entryType === 'mark' &&
```

Comment:
Renamed on line 90 above, because lint won't let you change the value of a parameter. Fixes this error:

  115:7  error  Assignment to function parameter 'entry'   no-param-reassign